python/lsst/ts/GenericCamera/driver/canoncamera.py

```diff
@@ -46,6 +50,10 @@ def __init__(self, log=None):
         # The path to the image in the camera
         self.file_path = None
 
+        # Add the Canon-related FITS header items to the generic ones.
+        fhig = FitsHeaderItemsGenerator()
```

Comment:
since you are not using this anywhere besides the append bellow, consider doing it all in one line:

```
self.tags.append(FitsHeaderItemsGenerator().generate_fits_header_items(FitsHeaderTemplate.CANON))
```

or, if you really want python to create an object just to append it later, add a more descriptive name, e.g.;

```
fits_header_generator = FitsHeaderItemsGenerator()
...
```

Reply:
I chose the first option.